src/rail/estimation/algos/deepdisc.py

```diff
@@ -167,44 +199,46 @@ def run(self):
         # optimizer = return_optimizer(cfg)
         optimizer = solver.build_optimizer(cfg_loader, model)
 
-        """
-        When using the single test dictionary, add this code and replace "mapper" below
-        
-        
+
+        #When using the single test dictionary, add this code and replace "mapper" below
         def dc2_key_mapper(dataset_dict):
```

Comment:
Let's move this function out of the body of the `run` function. If it's ever going to be reused, let's not keep it here. Alternatively, if it's not going to be reused, let's use a lambda function instead. 

Check with @grantmerz to determine if we'll need to reuse it.

Reply:
It will be reused in the estimator class.  We can move it to src/rail/deepdisc/

Reply:
Did this in https://github.com/LSSTDESC/rail_deepdisc/pull/20